PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-cognitiveservices/src/main/java/com/microsoft/azure/toolkit/intellij/cognitiveservices/CognitiveServicesStartupListener.java

```diff
@@ -51,16 +66,15 @@ public Object execute(@Nonnull Project project, @Nonnull Continuation<? super Un
                             .withLabel("Open in AI Playground")
                             .withSource(d)
                             .withHandler((_d, e) -> {
-                                am.getAction(OPEN_DEPLOYMENT_IN_PLAYGROUND).handle(d, e);
+                                AzureActionManager.getInstance().getAction(OPEN_DEPLOYMENT_IN_PLAYGROUND).handle(d, e);
```

Comment:
why don't handle null cases like L47 here?

Reply:
@Flanker32  no need, because user has logged in here.